R/mainTests.r

```diff
@@ -1153,6 +1160,454 @@ country_code <- function(data) {
   return(output)
 }
 
+
+# school_laestab_duplicates  ----------------------------------------
+# check that there is a 1:1 relationship between school laestab codes and names
+school_laestab_duplicates <- function(data) {
+  if (!"school_name" %in% names(data)) {
```

Comment:
Good point! Just added to the tests now - had a play around with removing URN and LAESTAB cols and the app displays a fail for the test instead of crashing now

Reply:
Nice! I'm still not sure about that the initial message saying there isn't any school level data, as that's only going by the presence of the school_name, would probably be more reliable to use `!"School" %in% unique(data$geographic_level)` as the first condition for ignoring, and then if school_name isn't there it would be a fail, like you've done for school_laestab (and school_urn)?

Reply:
Ah yes this makes much more sense thank you! Amended in the latest commit :)